ntp-daemon/Cargo.toml

```diff
@@ -2,6 +2,7 @@
 name = "ntp-daemon"
 version = "0.1.0"
 edition = "2021"
+license = "Apache-2.0/MIT"
```

Comment:
IIRC this is not a correct SPDX formulation, it should be something like "Apache-2.0 OR MIT".